packages/ember-htmlbars/tests/helpers/each_test.js

```diff
@@ -1,617 +0,0 @@
-    items: emberA(['one', 'two'])
-  });
-
-  runAppend(view);
-
-  assertHTML(view, 'onetwo');
-
-  run(function() {
-    view.set('items', emberA());
-  });
-
-  assertHTML(view, 'Nothing');
-});
-
-QUnit.test('it uses {{else}} when removing all items in an array', function() {
-  runDestroy(view);
-  var items = emberA(['one', 'two']);
-  view = EmberView.create({
-    template: compile('{{#each view.items as |item|}}{{item}}{{else}}Nothing{{/each}}'),
-    items
-  });
-
-  runAppend(view);
-
-  assertHTML(view, 'onetwo');
-
-  run(function() {
-    items.shiftObject();
-    items.shiftObject();
-  });
-
-  assertHTML(view, 'Nothing');
-});
-
-QUnit.test('it can move to and from {{else}} properly when the backing array gains and looses items (#11140)', function() {
-  runDestroy(view);
-  var items = emberA(['one', 'two']);
-  view = EmberView.create({
-    template: compile('{{#each view.items as |item|}}{{item}}{{else}}Nothing{{/each}}'),
-    items
-  });
-
-  runAppend(view);
-
-  assertHTML(view, 'onetwo');
-
-  run(function() {
-    items.shiftObject();
-    items.shiftObject();
-  });
-
-  assertHTML(view, 'Nothing');
-
-  run(function() {
-    items.pushObject('three');
-    items.pushObject('four');
-  });
-
-  assertHTML(view, 'threefour');
-
-  run(function() {
-    items.shiftObject();
-    items.shiftObject();
-  });
-
-  assertHTML(view, 'Nothing');
-});
-
-QUnit.module('{{each bar as |foo|}}', {
-  setup() {
-    owner = buildOwner();
-    owner.register('view:toplevel', EmberView.extend());
-  },
-  teardown() {
-    runDestroy(owner);
-    runDestroy(view);
-    owner = view = null;
-  }
-});
-
-QUnit.test('#each accepts a name binding', function() {
-  view = EmberView.create({
-    template: compile('{{#each view.items as |item|}}{{view.title}} {{item}}{{/each}}'),
-    title: 'My Cool Each Test',
-    items: emberA([1, 2])
-  });
-
-  runAppend(view);
-
-  equal(view.$().text(), 'My Cool Each Test 1My Cool Each Test 2');
-});
-
-QUnit.test('#each accepts a name binding and does not change the context', function() {
-  var controller = EmberController.create({
-    name: 'bob the controller'
-  });
-  var obj = EmberObject.create({
-    name: 'henry the item'
-  });
-
-  view = EmberView.create({
-    template: compile('{{#each view.items as |item|}}{{name}}{{/each}}'),
-    title: 'My Cool Each Test',
-    items: emberA([obj]),
-    controller: controller
-  });
-
-  runAppend(view);
-
-  equal(view.$().text(), 'bob the controller');
-});
-
-QUnit.test('#each accepts a name binding and can display child properties', function() {
-  view = EmberView.create({
-    template: compile('{{#each view.items as |item|}}{{view.title}} {{item.name}}{{/each}}'),
-    title: 'My Cool Each Test',
-    items: emberA([{ name: 1 }, { name: 2 }])
-  });
-
-  runAppend(view);
-
-  equal(view.$().text(), 'My Cool Each Test 1My Cool Each Test 2');
-});
-
-QUnit.test('#each accepts \'this\' as the right hand side', function() {
```

Comment:
[Migrated](https://github.com/emberjs/ember.js/pull/13349/commits/c08e75f3474e8bb995f5f24b837479bf3fca64f8#diff-d3c97c178f7e01e59143b5bf2f84062aR397) but not clear to me that this even makes sense in glimmer2 since everything is a component